book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] As it is a requirement for this project, please kindly add a method for adding rentals. The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association

Reply:
can u please help me with this issue cause It was really hard for me working on this project @Mihndim2020